jwst/associations/lib/rules_level2_base.py

```diff
@@ -175,17 +175,12 @@ def make_member(self, item):
         except KeyError:
             exposerr = None
 
-        # Times series and coronagraphic exposures
-        # process in integration space.
-        use_integrations = self.constraints['is_tso'].value == 't'
-        if not use_integrations:
-            use_integrations = item['exp_type'] in TSO_EXP_TYPES + CORON_EXP_TYPES
-
         # Create the member.
         member = Member(
             {
                 'expname': Utility.rename_to_level2a(
-                    item['filename'], use_integrations=use_integrations
+                    item['filename'],
+                    use_integrations=self.is_item_tso(item, other_exp_types=CORON_EXP_TYPES),
```

Comment:
OK, I see the intent here of classifying coronagraphic as TSO just for the sake of figuring out that they we need to use xxxints products for them, but casual reading is a bit confusing in that coronagraphic don't get processed as TSO.

Reply:
Commentary will be updated.